main.js

```diff
@@ -0,0 +1,65 @@
+let collection = [
+  {
+    title: 'Lorem ipsum',
+    author: 'Testeroo Testyy',
+  },
+  {
+    title: 'Second Book',
+    author: 'Testeroo Testyy',
+  },
+];
+
+function saveLocalStorage() {
+  localStorage.setItem('collection', JSON.stringify(collection));
+}
+
+function loadLocalStorage() {
+  if (JSON.parse(localStorage.getItem('collection'))) {
+    collection = JSON.parse(localStorage.getItem('collection'));
+  }
+}
+
+function addBook(newTitle, newAuthor) {
+  const newBook = {
+    title: newTitle,
+    author: newAuthor,
+  };
+  collection.push(newBook);
+}
+
+function removeBook(removeTitle) {
+  collection = collection.filter((book) => book.title !== removeTitle);
+}
```

Comment:
- [ ] Kindly check the functionality of the delete function. When I'm deleting a book that has the same name as another book and presses the button to delete on whatever both of them, both books are deleted. Where the book that needs to be deleted is the one that belongs to that button 🥇